packages/tenant-process/src/services/tenantService.ts

```diff
@@ -150,6 +151,29 @@ export async function retrieveAttribute(
   return attribute;
 }
 
+async function retrieveTenantAttribute(
+  tenantId: TenantId,
+  attributeId: AttributeId,
+  readModelService: ReadModelService
+): Promise<{ tenant: WithMetadata<Tenant>; attribute: Attribute }> {
+  const tenant = await retrieveTenant(tenantId, readModelService);
+  const attribute = await retrieveAttribute(attributeId, readModelService);
+  if (attribute.kind !== attributeKind.verified) {
+    throw attributeNotFound(attributeId);
+  }
+
+  const tenantAttribute = tenant.data.attributes.find(
+    (attr): attr is VerifiedTenantAttribute =>
+      attr.type === tenantAttributeType.VERIFIED && attr.id === attributeId
+  );
+
+  if (!tenantAttribute) {
+    throw attributeNotFoundInTenant(attributeId, tenantId);
+  }
```

Comment:
This throws three different errors: `tenantNotFound`, `attributeNotFound`, `attributeNotFoundInTenant` - let's add an error mapper in the router and map them to `404`, adding also corresponding API tests